Initial LVGL doc for ESPHome (#3678)
OttoWinter/esphomedocs: components/sensor/lvgl.rst

.. _lvgl-sns:

LVGL Sensor
===========

.. seo::

>     :description: Instructions for setting up a LVGL widget sensor component.

Reviewer: ```suggestion
    :description: Instructions for setting up an LVGL widget sensor component.
```

    :image: ../images/lvgl_c_num.png

> The ``lvgl`` sensor platform creates a semsor component from a LVGL widget

Reviewer: ```suggestion
The ``lvgl`` sensor platform creates a semsor component from an LVGL widget
```

and requires :ref:`LVGL <lvgl-main>` to be configured.

> Supported widgets are :ref:`lvgl-wgt-arc`, :ref:`lvgl-wgt-bar`, :ref:`lvgl-wgt-sli` and :ref:`lvgl-wgt-spb`. A single sensor supports a single widget, thus you need to choose among which one's state you want to use.

Reviewer: Maybe this?:
```suggestion
Supported widgets are :ref:`lvgl-wgt-arc`, :ref:`lvgl-wgt-bar`, :ref:`lvgl-wgt-sli` and :ref:`lvgl-wgt-spb`. A single sensor supports only a single widget; in other words, it's not possible to have multiple widgets associated with an ESPHome sensor.
```

Configuration options:
----------------------

- **id** (*Optional*, :ref:`config-id`): Manually specify the ID used for code generation.

> - **name** (**Required**, string): The name of the sensor.

Reviewer: ```suggestion
```
